src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService service;
+
+    @BeforeEach
+    void init() {
+        Storage.people.clear();
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registerValidUser_Ok() {
```

Comment:
```suggestion
    void register_validUser_ok() {
```
fix naming accordingly
use lowercase after underscore

Reply:
\+